ja: Update faq/netlify-deployment.md (#1446)
nuxt/docs: ja/faq/netlify-deployment.md

## 操作：

### 静的に生成されたサイトの場合

1. __ブランチをデプロイする：__ `master` もしくはデプロイしたいブランチ
2. __ビルドコマンド：__ `npm run generate`
3. __公開ディレクトリ：__ `dist`

> ## SPAモードで生成されたサイトの場合

Reviewer: ```suggestion
## SPA モードで生成されたサイトの場合
```

Please put a space between half-width characters and full-width characters. :pray:

1. __ブランチをデプロイする:__ `master` もしくはデプロイしたいブランチ
2. __ビルドコマンド:__ `npm run build`
3. __公開ディレクトリ:__ `dist`

> シングルページアプリケーションの場合、Netlify は再読み込み時にデフォルトで *"404 not found"* にリダイレクトしますが、[リダイレクト設定](https://www.netlify.com/docs/redirects/#rewrites)を行う事で防げます。 また、シングルページアプリケーションのヘッダーとリダイレクト両方の設定をサポートする[nuxt-netlify](https://www.bazzite.com/docs/nuxt-netlify)モジュールを利用する事もできます。

Reviewer: ```suggestion
シングルページアプリケーションの場合、Netlify は再読み込み時にデフォルトで *"404 not found"* にリダイレクトしますが、[リダイレクト設定](https://www.netlify.com/docs/redirects/#rewrites-and-proxying)を行う事で防げます。 また、シングルページアプリケーションのヘッダーとリダイレクト両方の設定をサポートする [nuxt-netlify](https://www.bazzite.com/docs/nuxt-netlify) モジュールを利用する事もできます。
```

+ Please put a space between half-width characters and full-width characters.
+ Fix the URL `https://www.netlify.com/docs/redirects/#rewrites` to `https://www.netlify.com/docs/redirects/#rewrites-and-proxying`

> > Divya Sasidharanが、Netlifyのリダイレクトに関するリファレンスを公開しているので参考にしてください。[blog](https://www.netlify.com/blog/2019/01/16/redirect-rules-for-all-how-to-configure-redirects-for-your-static-site)

Reviewer: ```suggestion
> Divya Sasidharan が、Netlify のリダイレクトに関するリファレンスを [blog](https://www.netlify.com/blog/2019/01/16/redirect-rules-for-all-how-to-configure-redirects-for-your-static-site) に公開しているので参考にしてください。
```
+ Please put a space between half-width characters and full-width characters.
+ How about insert `blog` in the sentence?
